habitat-lab/habitat/datasets/rearrange/rearrange_generator.py

```diff
@@ -544,6 +544,40 @@ def generate_single_episode(self) -> Optional[RearrangeEpisode]:
             targ_sampler_name_to_obj_sampler_names[
                 sampler_name
             ] = targ_sampler_cfg["params"]["object_samplers"]
+        dataset_name = osp.basename(self.cfg.dataset_path).split(".", 1)[0]
```

Comment:
Remove this, not sure why it's added here.

Reply:
Seems like it is used in the name of viewpoint cache

Reply:
Looks good.